samples/bot-adaptive-card-actions/csharp/AdaptiveCardActions/Startup.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Bot.Builder;
+using Microsoft.Bot.Builder.Integration.AspNet.Core;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+
+namespace Microsoft.BotBuilderSamples
+{
+    public class Startup
```

Comment:
Why are you using Startup.cs if it's already in .Net 6.0 ?

Reply:
We Should use program.cs

Reply:
I have used the bot suggested action, yes it was already there in .Net 6.0